src/observer/tests/test_json_rows.py

```diff
@@ -0,0 +1,38 @@
+import json
+from os.path import join
+from . import base
+from observer import utils, ingest_logic
+from django.test import Client
+from django.core.urlresolvers import reverse
+
+class One(base.BaseCase):
```

Comment:
`TestJsonResponse`|`TestJsonRowsIntegration` with a `test_published_research_article_index` method? Supports adding more examples for reproducing bugs